internal/resolver/dns/dns_resolver_test.go

```diff
@@ -742,45 +743,52 @@ func testDNSResolverWithSRV(t *testing.T) {
 	}()
 	defer leakcheck.Check(t)
 	tests := []struct {
-		target   string
-		addrWant []resolver.Address
-		scWant   string
+		target      string
+		addrWant    []resolver.Address
+		grpclbAddrs []resolver.Address
+		scWant      string
 	}{
 		{
 			"foo.bar.com",
 			[]resolver.Address{{Addr: "1.2.3.4" + colonDefaultPort}, {Addr: "5.6.7.8" + colonDefaultPort}},
+			nil,
 			generateSC("foo.bar.com"),
 		},
 		{
 			"foo.bar.com:1234",
 			[]resolver.Address{{Addr: "1.2.3.4:1234"}, {Addr: "5.6.7.8:1234"}},
+			nil,
 			generateSC("foo.bar.com"),
 		},
 		{
 			"srv.ipv4.single.fake",
-			[]resolver.Address{{Addr: "2.4.6.8" + colonDefaultPort}, {Addr: "1.2.3.4:1234", Type: resolver.GRPCLB, ServerName: "ipv4.single.fake"}},
+			[]resolver.Address{{Addr: "2.4.6.8" + colonDefaultPort}},
+			[]resolver.Address{{Addr: "1.2.3.4:1234", Type: resolver.Backend, ServerName: "ipv4.single.fake"}},
```

Comment:
If you change the resolver to not explicitly set `Backend`, change the the tests, too.

Reply:
Done